src/main/java/lotto/domain/LottoNumbers.java

```diff
@@ -0,0 +1,69 @@
+package lotto.domain;
+
+import lotto.exception.IllegalLottoNumberCountException;
+import lotto.exception.IllegalSeparateException;
+
+import java.util.*;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class LottoNumbers {
+
+    private static final Pattern SEPARATE_PATTERN = Pattern.compile(",");
+    private final Set<LottoNumber> lottoNumbers;
+
+    public LottoNumbers(List<Integer> lottoNumbers) {
+        this(toLottoNumbers(lottoNumbers));
+    }
+
+    public LottoNumbers(Set<LottoNumber> lottoNumbers) {
+        lottoNumberCountValidate(lottoNumbers);
+        this.lottoNumbers = lottoNumbers;
+    }
```

Comment:
```suggestion
    public LottoNumbers(Set<LottoNumber> lottoNumbers) {
        lottoNumberCountValidate(lottoNumbers);
        this.lottoNumbers = lottoNumbers;
    }
    
    public LottoNumbers(List<Integer> lottoNumbers) {
        this(toLottoNumbers(lottoNumbers));
    }
```
이렇게 순서를 바꿔주면 더 좋을 것 같아요~